window_manager/include/core/XWWindow.hpp

```diff
@@ -4,11 +4,19 @@
 #include <memory>
 #include <future>
 
+#include <boost/log/trivial.hpp>
```

Comment:
Same as above, not used inside header, so should be relocated to source if used there, otherwise removed.